### ISSUE TITLE
LC: Pomeg Glitch moves & Gen IV abilities

### ISSUE THREAD

**Issue by Antar1011** [smogon/pokemon-showdown#146]
Genderless Pokemon in Little Cup have vastly limited movepools (limited to only TM, HM, Move Tutor moves, and whatever few moves are available by Lv. 5), however if the Pokemon is available from the ADV era, the Pomeg glitch can be exploited to give the Pokemon ANY Level-Up move.

That's all well and good... except for genderless Pokemon who had only one ability prior to Gen IV (apparently this is just Porygon). Basically, since that Pokemon can only have those Level-up moves by being bred in Gen III, it can't have the Gen IV ability.

tl;dr--LC Porygon MUST have Trace if it has any level-up moves beyond Conversion(2), Sharpen and Tackle.

In addition, it looks like genderless level-up restrictions aren't being enforced for Pokemon only obtainable in Gen IV/V, namely I'm seeing Golett with DynamicPunch.
